scripts/backup

```diff
@@ -30,6 +30,8 @@ db_name=$(ynh_app_setting_get $app db_name)
 #=================================================
 # BACKUP APP MAIN DIR
 #=================================================
+# Clean cache files before backup (saved some disk space)
+ynh_secure_remove --file=${final_path}/var/cache/prod/
```

Comment:
Sure that to remove the directory and not recreate it is ok for wallabag ?

Reply:
Well, even when removing this folder Yunohost raise that warning : `Warning: Avoid deleting /var/www/wallabag2/var/cache/prod/` :sweat_smile:

But backup+restore works perfectly. And the directory is recreated by wallabag.

ping @nicosomb @j0k3r @tcitworld : do you know if removing the cache folder can cause trouble to Wallabag ?

Reply:
If the cache is removed, it'll be re-created on the next request to wallabag. That request will be a bit slower but that shouldn't be visible to the end user.

Reply:
Perfect, as I expected. Even if it was slower it's not a big deal, it's only after a backup.

Reply:
To avoid this warning message, remote the last `/`.

`ynh_secure_remove --file=${final_path}/var/cache/prod`

Like this this folder will be removed

Reply:
Done :)